src/test/java/FlightsReporterTests.groovy

```diff
@@ -0,0 +1,197 @@
+import org.junit.Before
+import org.junit.Test
+import org.junit.Assert
+import org.junit.jupiter.api.BeforeAll
+
+import java.nio.channels.FileLock
+import java.time.LocalDateTime
+import java.time.LocalTime;
+//import FlightsReporter
+
+import static  org.junit.jupiter.api.Assertions.*;
+
+class FlightsReporterTests {
+    List<Flight> flights = []
+
+    @Test
+    void testFlightLengthInMinutes() {
+        Flight fl1 = new Flight(id: "SOFLON11", from: FlightsReporter.Airports.SOFIA, destination: FlightsReporter.Airports.LONDON,
+            departure: LocalDateTime.of(2021, 8, 3, 10, 30),
+                arrival: LocalDateTime.of(2021, 8, 3, 13, 15),
+                passengers: 100, capacity: FlightsReporter.CAPACITIES[3], runningLate: false)
+
+        assertEquals(FlightsReporter.flightLengthInMinutes(fl1), 165)
+
+    }
+
+    @Test
+    void testGetAllFlightsTo() {
+        FlightsReporter.printFlights(flights)
+        assert FlightsReporter.getAllFlightsTo(flights, FlightsReporter.Airports.AMSTERDAM)
```

Comment:
You can use the groovy == to compare lists. 
Arrays.asList() is way more ugly as seen in the line below